visionSamples/barcode-reader/app/src/main/java/com/google/android/gms/samples/vision/barcodereader/BarcodeTrackerFactory.java

```diff
@@ -25,16 +27,19 @@
  * multi-processor uses this factory to create barcode trackers as needed -- one for each barcode.
  */
 class BarcodeTrackerFactory implements MultiProcessor.Factory<Barcode> {
-    private GraphicOverlay<BarcodeGraphic> mGraphicOverlay;
+    private GraphicOverlay<BarcodeGraphic> graphicOverlay;
+    private Context context;
```

Comment:
I think you accidentally renamed the mGraphicOverlay variable. Can you change it back and also rename context to mContext to be consistent with the rest of the code?

Reply:
Done.